src/main/java/core/basesyntax/Calculator.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Calculator {
+    public double calculate(int a, int b, char operate) {
```

Comment:
```suggestion
    public double calculate(int a, int b, char operator) {
```
also, consider renaming `a` and `b` - this is not very informative

Reply:
done